cvc5_z3py_compat/z3.py

```diff
@@ -7399,8 +7399,8 @@ def sort(self):
 
 def TupleSort(name, sorts, ctx=None):
     """Create a named tuple sort base on a set of underlying sorts
-    Example:
-        >>> pair, mk_pair, (first, second) = TupleSort("pair", [IntSort(), BoolSort()])
+
```

Comment:
It may be good to document what this method returns...

Reply:
Agreed.